genesyscloud/resource_exporter/resource_name_sanitizer.go

```diff
@@ -112,63 +99,6 @@ func (sod *sanitizerOriginal) SanitizeResourceHash(originalBlockLabel string) st
 	return strconv.FormatUint(uint64(algorithm.Sum32()), 10)
 }
 
-// Sanitize sanitizes all resource label using the time optimized algorithm
-func (sod *sanitizerOptimized) Sanitize(idMetaMap ResourceIDMetaMap) {
-	sanitizedLabels := make(map[string]int, len(idMetaMap))
-
-	for _, meta := range idMetaMap {
-		sanitizedLabel := sod.SanitizeResourceBlockLabel(meta.BlockLabel)
-
-		if meta.OriginalLabel == "" {
-			meta.OriginalLabel = meta.BlockLabel
-		}
-
-		if sanitizedLabel != meta.BlockLabel {
-			if count, exists := sanitizedLabels[sanitizedLabel]; exists {
-				// We've seen this sanitized label before
-				sanitizedLabels[sanitizedLabel] = count + 1
-
-				// Append a hash to ensure uniqueness
-				hash := sod.SanitizeResourceHash(meta.BlockLabel)
-				meta.BlockLabel = sanitizedLabel + "_" + hash
-				if meta.OriginalLabel == "" {
-					meta.OriginalLabel = meta.BlockLabel
-				}
-
-			} else {
-				sanitizedLabels[sanitizedLabel] = 1
-				meta.BlockLabel = sanitizedLabel
-			}
-		}
-	}
-}
-
-// SanitizeResourceBlockLabel sanitizes a single resource label
-func (sod *sanitizerOptimized) SanitizeResourceBlockLabel(inputLabel string) string {
```

Comment:
Please pull this implementation in to the original sanitizer. It adds useful transliteration of the block labels (so Japanese characters become ASCII character equivalents).

Reply:
Done. I pulled the optimized sanitizer's SanitizeResourceHash method back in also